admin/database/postgres/migrate.go

```diff
@@ -94,12 +93,12 @@ func (c *connection) Migrate(ctx context.Context) (err error) {
 		if err != nil {
 			return err
 		}
-		defer tx.Rollback()
+		defer func() { _ = tx.Rollback() }()
```

Comment:
What linting rule is this? Seems unnecessary

Reply:
its `errcheck` i think we should not ignore this linter, else we can't catch err checked missed by anyone